.github/workflows/ci.yml

```diff
@@ -18,7 +39,7 @@ jobs:
           -G "Unix Makefiles" \
           -DCMAKE_BUILD_TYPE=Debug \
           -DBUILD_CLONE_SUBMODULES=ON \
-          -DCMAKE_C_FLAGS='--coverage -Wall -Wextra -Werror -DNDEBUG'
+          -DCMAKE_C_FLAGS='--coverage -Wall -Wextra -Werror -DNDEBUG -Wno-error=pedantic -Wno-variadic-macros -DLOGGING_LEVEL_DEBUG=1'
```

Comment:
Why are these additional arguments needed for building unit tests and coverage but not for the build checks?

Reply:
These warnings only appear for the variadic macro-based implementation of the logging macros provided in the `core_sntp_config.h` file for unit tests.
In the library build, which uses the `coverity_analysis` target, logging is disabled as the `SNTP_DO_NOT_USE_CUSTOM_CONFIG` flag is passed